WordPress/openverse
# Change the header height to 5rem(80px)

## Fixes



Related to #1159 by @obulat

## Description


To make the actual change in #2128 easier to review, I extracted the change to the header height here.
This PR also adds the visual regression test for banners to clearly show the problem that is solved in #2128.

Mixing up rem and px might cause a problem if the user's default font in the browser is not 16px, this is why I used `rem` for the header height.

## Testing Instructions

Everything should work as usual and the CI should pass. The only difference in this PR is the header that is 1px smaller than currently: 80px (5rem) instead of 81px. 

Check out the new snapshots with all the banners to see that currently the top of the filters sidebar is covered when both banners are shown: https://github.com/WordPress/openverse/blob/b60fa265f13dc4b42cd8a6c6322075551548a2b6/frontend/test/playwright/visual-regression/pages/search-with-banners.spec.ts-snapshots/page-with-all-banners-2xl-linux.png

## Checklist



[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
